index.js

```diff
@@ -0,0 +1,98 @@
+//DOM ELEMENTS
+const playerImg = document.querySelector(".player img")
+const h2Name = document.querySelector(".player h2")
+const emNickname = document.querySelector(".player em")
+const pLikes = document.querySelector(".player p")
+const goalsUl = document.querySelector("#goals")
+const likeButton = document.querySelector(".like-button")
+const goalForm = document.querySelector("#new-goal-form")
+
+
+//FETCH DATA
+function getData() {
+    return fetch("http://localhost:3000/players/1")
+        .then(resp => resp.json())
+        .then(playerObj => renderPlayer(playerObj))
+}
+getData()
+
+//Event Listeners
+
+likeButton.addEventListener("click", getLikes)
+goalForm.addEventListener("submit", newGoal)
+
+//Event Callbacks
+
+function newGoal(event) {
+    event.preventDefault()
+    const newGoalObj = {
+        // id: 5,
+        playerId: parseInt(event.target.previousElementSibling.dataset.id),
```

Comment:
Here, it may have been easier to just add the `data-id` directly to the form element instead, so you could just do `event.target.dataset.id`.